### PR TITLE
RN 12.3.1 to Preview

### PR DESCRIPTION
**Pull request by albertotessarotto**
## Pull Request Type

- [ ] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [ ] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [ ] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [ ] No sensitive content has been committed

### CODE REVIEW

File: docs/release-notes/v12.3.1.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+### Marketplace Updates
+
+#### Authentication Service
+
+The new version `v3.9.0` of the Authentication Service is available!
+
+##### Improvements
+
+Added a new endpoint to the authentication service that exposes the stored `idToken`. The endpoint can be exposed only for the provider `microsoft` through a dedicated setting. For more information consult the documentation.
```
Comment (malta895) [inline docs/release-notes/v12.3.1.md:85]:
```suggestion
Added a new endpoint to the authentication service that exposes the stored OIDC provider `idToken`. 
The endpoint can be exposed **only for the provider `microsoft`** through a dedicated setting. 
For more information consult the [service documentation](/runtime_suite/authentication-service/30_usage.md#get-provider-idtoken).
```

File: docs/release-notes/v12.3.1.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+### Fast Data Configurator
+
+#### Bug Fix
+
+This version addressed a few bugs, here is a comprehensive list:
+
+* Single View indexes marked as "default sorting" are now **not** deletable
+
+### Fast Data Services
```
Comment (albertotessarotto) [inline docs/release-notes/v12.3.1.md:46]:
```suggestion
```

File: docs/release-notes/v12.3.1.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+#### Improvements
+
+##### Search bar in layout
+
+It is now possible to search between menu items in `bk-layout` component. Moreover, the collapse button has a new style.
+
+##### Estimate count support
+
+`bk-crud-client` has the new property `useEstimateCount` that enables to do every `/count` call with the `_useEstimateCount` query parameter set to true. Moreover, `bk-pagination` has the new property `usePagesCount` to hide the pages count when `false`.
+
```
Comment (albertotessarotto) [inline docs/release-notes/v12.3.1.md:65]:
```suggestion
```

File: docs/release-notes/v12.3.1.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+#### Bug Fix
+
+This version addressed a few bugs, here is a comprehensive list:
+
+* improved behavior of `keepPageCount` in `bk-crud-client`
+* fixed files preview in `bk-file-client`
+
```
Comment (albertotessarotto) [inline docs/release-notes/v12.3.1.md:73]:
```suggestion
```

File: docs/release-notes/v12.3.1.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+### Improvements
+
+#### Last Triggered Event in Webhook Section
+
+A new column has been added to the webhooks table within the Console's Webhook section, enabling the display of the date, status, and event type of the most recent triggering of the specifically configured webhook.
+
+#### Collapse and expand nodes and events in Flow Manager Configurator
+
+A new button has been added to the Flow Manager Configurator, enabling the programmatic expand an collapse of nodes and events labels for a better navigability of flows.
```
Comment (albertotessarotto) [inline docs/release-notes/v12.3.1.md:28]:
```suggestion
A new button has been added to the [Flow Manager Configurator](/development_suite/api-console/api-design/flow-manager-configurator/overview.md), enabling the programmatic expand an collapse of nodes and events labels for a better navigability of flows.
```

File: docs/release-notes/v12.3.1.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+##### Improvements
+
+Added a new endpoint to the authentication service that exposes the stored OIDC provider `idToken`. 
+The endpoint can be exposed **only for the provider `microsoft`** through a dedicated setting. 
+For more information consult the [service documentation](/runtime_suite/authentication-service/30_usage.md#get-provider-idtoken).
+
+## How to update your Console
+
+For self-hosted installations, please head to the [self hosted upgrade guide](/infrastructure/self-hosted/installation-chart/100_how-to-upgrade.md#v12---version-upgrades) or contact your Mia-Platform referent and upgrade to _Console Helm Chart_ `vX.X.X`.
```
Comment (albertotessarotto) [inline docs/release-notes/v12.3.1.md:91]: to insert correct helm chart

File: docs/release-notes/v12.3.1.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+* it has been fixed a bug that wrongly inserted a variable that needed interpolation when using the swagger-aggregator with the Enhanced Project Workflow.
+
+## Fast Data
+
+### Fast Data Configurator
+
+#### Bug Fix
+
+This version addressed a few bugs, here is a comprehensive list:
```
Comment (marcofilippi) [inline docs/release-notes/v12.3.1.md:42]:
```suggestion
This version addressed the following bug:
```

**Review by marcofilippi — COMMENTED** [review]
typo fix

**Review by jacoposte — APPROVED** [review]

**Review by marcofilippi — APPROVED** [review]

**Pull request merged**